ccloud/beginner-cloud/start.sh

```diff
@@ -271,20 +270,14 @@ ccloud kafka acl delete --allow --service-account $SERVICE_ACCOUNT_ID --operatio
 ##################################################
 
 TOPIC3="demo-topic-3"
-
+CONNECTOR="datagen_ccloud_pageviews"
```

Comment:
Maybe also update this comment from:

```
##################################################
# Run Connect and kafka-connect-datagen connector with permissions
# - Confluent Hub: https://www.confluent.io/hub/
##################################################		 
```

to something along the lines of

```
##################################################
# Run the fully-managed connector kafka-connect-datagen in Confluent Cloud
##################################################		 
```

Reply:
Corrected.